c_common/front_end_common_lib/local.mk

```diff
@@ -121,15 +112,8 @@ OBJECTS += $(_OBJS)
 
 include $(SPINN_DIRS)/make/spinnaker_tools.mk
 
-$(APP_DICT_FILE): $(LOG_DICT_FILES)
-	@echo mklogdict -o $(APP_DICT_FILE) $(LOG_DICT_FILES)
-    # Add the two header lines once
-	@head -2 $(firstword $(LOG_DICT_FILES)) > $(APP_DICT_FILE)
-    # Add the none header lines for each file remembering tail starts counting at 1
-	@$(foreach ldf, $(LOG_DICT_FILES), tail -n +3 $(ldf) >> $(APP_DICT_FILE) ;)
-
 # Tidy and cleaning dependencies
 clean:
-	$(RM) $(TEMP_FILES) $(OBJECTS) $(BUILD_DIR)$(APP).elf $(BUILD_DIR)$(APP).txt $(ALL_TARGETS) $(LOG_DICT_FILE) $(BUILD_DIR)$(APP).nm $(BUILD_DIR)$(APP).elf $(BUILD_DIR)$(APP).bin
+	$(RM) $(TEMP_FILES) $(OBJECTS) $(BUILD_DIR)$(APP).elf $(BUILD_DIR)$(APP).txt $(ALL_TARGETS) $(BUILD_DIR)$(APP).nm $(BUILD_DIR)$(APP).elf $(BUILD_DIR)$(APP).bin
```

Comment:
like this line

Reply:
Yes, Makefile is harder to stick to 80 characters without making "a right mess" (technical term)